pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/blocks/TransferableBlock.java

```diff
@@ -188,4 +193,10 @@ private boolean isType(MetadataBlock.MetadataBlockType type) {
     MetadataBlock metadata = (MetadataBlock) _dataBlock;
     return metadata.getType() == type;
   }
+
+  @Override
+  public String toString() {
+    String blockType = isErrorBlock() ? "error" : isSuccessfulEndOfStreamBlock() ? "eos" : "data";
+    return "TransferableBlock{blockType=" + blockType + ", _numRows=" + _numRows + ", _container=" + _container + '}';
```

Comment:
We may want to improve the `toString`, but it is very useful in logs and right now it doesn't print customer information

Reply:
Removed in 55d9513640904c5651be88b06bc7a99db5a57c62